packages/core/types/index.js

```diff
@@ -257,31 +257,35 @@ export type InitialParcelOptions = {|
   +env?: EnvMap,
   +targets?: ?(Array<string> | {+[string]: TargetDescriptor, ...}),
 
-  +disableCache?: boolean,
+  +disableCache?: boolean, // TODO: Change to shouldDisableCache or shouldEnableCache?
```

Comment:
Not sure what to do with this.

Should it become `shouldDisableCache` or `shouldEnableCache`

Reply:
IMO `shouldDisableCache` given the default is enabled. I think we should try to keep boolean options to have the default be `false` if possible.